azure-resources/Compute/virtualMachines/kql/f0a97179-133a-6e4f-8a49-8a44da73ffce.kql

```diff
@@ -1,2 +1,11 @@
-// under-development
-
+// Azure Resource Graph Query
+// Provides a list of Azure VM disks without Azure Disk Encryption or encryption at host enabled
+resources
+| where type == "microsoft.compute/disks"
+| project diskId = id, diskName = name, vmId = tolower(managedBy), azureDiskEncryption = iff(properties.encryptionSettingsCollection.enabled == true,'Enabled','Disabled')
```

Comment:
- We can use Boolean instead of 'Enabled' and 'Disabled'.

```suggestion
| project diskId = id, diskName = name, vmId = tolower(managedBy), azureDiskEncryption = iff(properties.encryptionSettingsCollection.enabled == true, true, false)
```